qa/integration/specs/pipeline_log_spec.rb

```diff
@@ -112,6 +113,32 @@
     expect(plaing_log_content =~ /Pipeline started {"pipeline.id"=>"#{pipeline_name}"}/).to be_nil
   end
 
+  it "should rollover main log file when pipeline.separate_logs is enabled" do
+    pipeline_name = "custom_pipeline"
+    settings = {
+      "path.logs" => temp_dir,
+      "pipeline.id" => pipeline_name,
+      "pipeline.separate_logs" => true
+    }
+    FileUtils.mkdir_p(File.join(temp_dir, "data"))
+    data = File.join(temp_dir, "data")
+    settings = settings.merge({ "path.data" => data })
+    IO.write(File.join(temp_dir, "logstash.yml"), YAML.dump(settings))
+
+    FileUtils.cp("fixtures/logs_rollover/log4j2.properties", temp_dir)
```

Comment:
seems there isn't that much difference in the fixture properties file, 
would it be feasible to generate a .properties based on actual `config/log4j2.properties` here 
and add/replace lines - to make sure we stay in sync?

possible future changes to *config/log4j2.properties* might cause we end up testing a non-production config ...

Reply:
Good point, I think we could have a good compromise with a Gradle task that copies the production `łog4j2.properties` and test against that

Reply:
after it had changed the size of the rollover part